terraform/modules/cos/variables.tf

```diff
@@ -119,3 +120,22 @@ variable "tempo_query_frontend_units" {
   type        = number
   default     = 3
 }
+
+
+variable "cloud" {
+  description = "Kubernetes cloud or environment where this COS module will be deployed (e.g self-managed, aws)"
+  type        = string
+  default     = "self-managed"
```

Comment:
Should these values map 1:1 to the output of `juju clouds --all`? In that case `self-managed` should be changed to localhost?